docs/content/manual/virtual-machines/delete-single-vm-all-disks.md

```diff
@@ -1,12 +1,14 @@
 ---
 title: Delete single vm all disks (e2e_be)
 ---
-1. Delete VM
-1. Select whether you want to delete disks
+1. Create a VM
+1. Make sure VM have started in running state with IP address
+1. Delete the VM
+1. Select the option `Select the volume you want to delete` (delete volume)
 
 ## Expected Results
 1. You should check amount of used space on Server before you delete the VM
-1. Machine should delete
+1. VM should be deleted
 1. It should not show up in the Virtual Machine list
-1. Disks should be listed/or not in Volumes list as appropriate
+1. All volumes attached to the VM should be deleted on the volume psage
```

Comment:
typo

Reply:
Thanks for the check. Update the typo to volume page.